src/test/java/org/apache/sysds/test/component/frame/FrameRemoveEmptyTest.java

```diff
@@ -35,7 +35,7 @@
 	private final static String TEST_DIR = "functions/frame/";
 	private static final String TEST_CLASS_DIR = TEST_DIR + RemoveEmptyTest.class.getSimpleName() + "/";
 
-	private final static int _rows = 10;
+	private final static int _rows = 100;
```

Comment:
add tests for selecting for selecting 0, 1, half, all-but-one, all.
in case of selecting all, simply return a deep copy.

100 does not really change performance measurements.
while debugging try to increase it, same goes for columns.

then create measurements for selecting 1, half, all-but-one elements from the input.
and find out if the new version is faster or slower in each case.

In the PR message argue the performance difference say something like 2x faster etc.